test/tools/CMakeLists.txt

```diff
@@ -33,5 +33,11 @@ add_evmc_tool_test(
     "Result: +success[\r\n]+Gas used: +7[\r\n]+Output: +aabbccdd00000000000000000000000000000000000000000000000000000000[\r\n]"
 )
 
+add_evmc_tool_test(
+    create_return_2
```

Comment:
Nit: why the _2 suffix?

Reply:
There is also `create_return_1` unit tests.